net-misc/cadaver/cadaver-0.23.3.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="A command line WebDAV client"
+DESCRIPTION="It supports file upload, download, on-screen display, namespace operations (move/copy), collection creation and deletion, and locking operations."
+HOMEPAGE="http://www.webdav.org/cadaver"
+COPYRIGHT="1999-2007, Joe Orton, 
+   except where otherwise indicated."
+LICENSE="GNU GPL v2"
+REVISION="3"	
+SOURCE_URI="http://www.webdav.org/cadaver/cadaver-0.23.3.tar.gz"
+CHECKSUM_SHA256="fd4ce68a3230ba459a92bcb747fc6afa91e46d803c1d5ffe964b661793c13fca"
+#PATCHES=""
+
+ARCHITECTURES="x86 x86_64"
```

Comment:
Add x86_gcc2 in here (it builds ok with gcc2)

Reply:
I've just added a comment in the wiki with the preferred order:
https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes#architectures